### ISSUE TITLE
unable to see features stored in the feature-store for LTR in docker-solr instance

### ISSUE THREAD

**Issue by prachijain78** [docker-solr/docker-solr#335]
I am trying to work with LTR in docker-solr and I am not able to get past the step about uploading features to feature-store. I need help to understand what i might be doing wrong. Is there a way to check that the plugin is enabled successfully? Is there any other way to upload features and model? Do i need to do things differently in solr that is run in docker container? My ultimate goal is to have efi features setup for LTR

Below are the details of the steps I followed and the setup done.

I followed the instructions and did all the steps to set up feature and model as specified here http://mirror.olnevhost.net/pub/apache/lucene/solr/ref-guide/apache-solr-ref-guide-8.1.pdf (pg-594 onwards) including the changes to solrconfig.xml. I enabled the LTR plugin passing the option ` -e SOLR_OPTS="-Dsolr.ltr.enabled=true" `in my docker run command.

The features that i had in the features1.json file were as below
`
[ 
  {
    "store" : "my_feature_store",
    "name" : "original_score",
    "class" : "org.apache.solr.ltr.feature.OriginalScoreFeature",
    "params" : {}
  }
]
`
and model1.json looked like
`
{
  "store" : "my_feature_store",
  "name" : "my_model",
  "class" : "org.apache.solr.ltr.model.LinearModel",
  "features" : [    
    { "name" : "original_score" }
  ],
  "params" : {
    "weights" : {
      "original_score" : 1.0
    }
  }
}
`
I uploaded features to solr using
`
curl -XPUT 'http://localhost:8983/solr/test/schema/feature-store' --data-binary "@/path/to/json/features1.json" -H 'Content-type:application/json'
`
After that i reloaded the core (since i Understand this being a managed resource) using
`
curl -u admin:JjZs5x4fp6cc "http://localhost:8983/solr/admin/cores?action=RELOAD&core=test"
`
I do not see any errors on running any of the commands or in the logs but when i check the features stored using the url http://localhost:8983/solr/test/schema/feature-store/my_feature_store in browser I get an empty list, i see a response like
`
{
  "responseHeader":{
    "status":0,
    "QTime":2},
  "features":[]}
`
I tried with different features, like I tried with documentRecency that is given in the examples in the pdf mentioned above, i tried using  `_DEFAULT_ ` store by eliminating the store name from the json file but nothing seems to work.

**Comment by beviah** [comment]
Indeed, having the same issue.. kind of frustrating given there were no error messages during loading, nor in logs.

**Comment by tkaessmann** [comment]
Hey,
we run successfully LTR based on solr-docker by following the steps from the documentation. Do you have documents available in your solr?
Maybe for your problem it's better to use the [slack channel or the solr mailing list](https://solr.apache.org/community.html).
